index.html

```diff
@@ -61,7 +61,7 @@ <h3>My Recent Works</h3>
                 <hr>
             </div>
             <div class="recent-work">
-                <a href=""><img src="./assets/Img_Placeholder_1.png" alt="Women practicing yoga"></a>
+                <a href=""><img class="recent-work-img" src="./assets/Img_Placeholder_1.png" alt="Women practicing yoga"></a>
                 <div>
                     <h4>Multi-Post Stories</h4>
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] Let's add a popup window for this card here, as there is no popup for it currently. This is to ensure that all project cards have a popup window and are rendered dynamically using JavaScript.